lib/active_merchant/billing/gateways/cyber_source_rest.rb

```diff
@@ -94,13 +98,52 @@ def add_amount(post, amount)
         }
       end
 
-      def add_credit_card(post, creditcard)
+      def add_payment(post, payment, options)
+        if payment.is_a?(NetworkTokenizationCreditCard)
+          add_apple_google(post, payment, options)
+        else
+          add_credit_card(post, payment)
+        end
+      end
+
+      def add_apple_google(post, payment, options)
+        post[:processingInformation] = {
+          paymentSolution: PAYMENT_SOLUTION[payment.source]
+        }
+        # supported cards: JCB, VISA, MASTERCARD
+        post[:processingInformation][:commerceIndicator] = 'internet' unless card_brand(payment) == 'jcb'
```

Comment:
💬 as far I can see on the docs, looks like the  `commerceIndicator` field doesn't is necessary for this transaction, and instead is useful for stored credentials, and also the validation is confusing according to the comment

Reply:
The commerceIndicator is required for MasterCard transactions (without the 3ds) and Visa transactions. There are three supported card types for Apple/Google Pay transactions (JCB, Visa, and Mastercard).

Reply:
ah ok, but why this validation `unless card_brand(payment) == 'jcb'` ? If `jcb` is one of the card types supported

Reply:
it's because 'internet' is only required for visa an mc